source/paramgen.go

```diff
@@ -75,6 +81,12 @@ func (Config) Parameters() map[string]config.Parameter {
 			Type:        config.ParameterTypeDuration,
 			Validations: []config.Validation{},
 		},
+		ConfigRetries: {
+			Default:     "0",
```

Comment:
Readme says default value is `1000`. We need to consolidate. Either, the code in config or readme.

Reply:
updated.